### PR TITLE
feat: improvement of Profile section

### PR DESCRIPTION
**Pull request by rokunski**
## Proposed Changes
Profile section rework:
* almost all of missing elements/attributes explained 
* some layout changes

## Additional informations
*Check, if it is ok or it need some rework.*

### CODE REVIEW

File: profile.md [hunk 1]
```diff
@@ -11,37 +11,144 @@ This section of the documentation contains things, that can't be described using
...
-The root element of the profile is the profile. It takes two attributes.
+The root element of the profile is the profile. It takes up to five attributes.
+
+| Name    | Type     | Allowed values | Description                                           | Optional |
+| ------- | -------- | -------------- | ----------------------------------------------------- | -------- |
+| id      | String   |                | A profile name                                        |          |
+| side    | String   | left, right    | Placement in terms of refx                            | True     |
+| maxstep | +Float   |                | A profile linear points max interval length           | True     |
+| maxturn | +Float   |                | A profile nonlinear points max interval length        | True     |
+| refx 	  | String   |                | A reference to the X-axis, which profile is based on  | True     |
```
Comment (Mensix) [inline profile.md]: Formatting

File: profile.md [hunk 2]
```diff
@@ -11,37 +11,144 @@ This section of the documentation contains things, that can't be described using
...
 It's not usually required to adjust the maxstep. One of cases when you can adjust it might be better looking spot lights.
 :::
 
-| Name    | Type     | Allowed values | Description                         | Optional |
-| ------- | -------- | -------------- | ----------------------------------- | -------- |
-| id      | String   |                | A profile name                      |          |
-| side    | String   | left, right    | Placement in terms of refx          | True     |
-| maxstep | +Integer | 1000           | A profile linear points split count | True     |
+::: tip
+Maxturn attribute can be understood as max chord length of curved lines. The smaller the maxturn value is, the smoother the profile is, but more vertices and faces are rendered. Default value is circa 0.5.
```
Comment (Mensix) [inline profile.md]: No default value is specified.

File: profile.md [hunk 3]
```diff
@@ -11,37 +11,144 @@ This section of the documentation contains things, that can't be described using
...
+| y     | Float  | The next point in position relative to the Y-axis                          |          |
+| k1    | Float  | First derivative at the start of polynom3                                  | True     |
+| k2    | Float  | First derivative at the end of polynom3                                    |          |
+| refx  | String | A reference to the X-axis, which profile is based on                       | True     |
+| refy  | String | A reference to the Y-axis, which profile is based on                       | True     |
+| refyx | Float  | A reference to X-point, which Y-point (from *y* attribute) is based on     | True     |
+
+::: tip
+The start of polynom3 means the end point of previous element.  
+:::
```
Comment (Mensix) [inline profile.md:95]: Enter key between tips

File: profile.md [hunk 4]
```diff
@@ -11,37 +11,144 @@ This section of the documentation contains things, that can't be described using
...
+| x     | Float  | The next point in position relative to the X-axis                          |          |
+| y     | Float  | The next point in position relative to the Y-axis                          |          |
+| x0    | Float  | The centre point of circle in position relative to X0-axis                 |          |
+| y0    | Float  | The centre point of circle in position relative to Y0-axis                 |          |
+| refx  | String | A reference to the X-axis, which profile is based on                       | True     |
+| refy  | String | A reference to the Y-axis, which profile is based on                       | True     |
+| refyx | Float  | A reference to X-point, which Y-point (from *y* attribute) is based on     | True     |
+
+::: warning 
+X0(Y0)-axis is the axis of global coordinates system. You can't change its reference.
```
Comment (Mensix) [inline profile.md]: [x, y]0-axis are the axis of global coordinates system. You can't change its reference.

**Review by Mensix — COMMENTED** [review]
👍

**Comment by rokunski** [comment]
Add these changes as you wanted.

**Pull request merged**